package.json

```diff
@@ -26,6 +26,7 @@
     "@openzeppelin/test-helpers": "^0.5.15",
     "@uniswap/v2-core": "^1.0.1",
     "@uniswap/v2-periphery": "^1.1.0-beta.0",
+    "axios": "^0.25.0",
```

Comment:
Remove axios and use note-fetch instead

Reply:
This dependency and everything related to scripts will be removed from the code, since now it's on the CLI codebase (using node-fetch there).